test/SpokePool.Admin.ts

```diff
@@ -24,6 +24,19 @@ describe("SpokePool Admin Functions", async function () {
     expect(await spokePool.depositQuoteTimeBuffer()).to.equal(60);
   });
 
+  it("Pause deposits and fills", async function () {
+    expect(await spokePool.pausedDeposits()).to.equal(false);
+    expect(await spokePool.pausedFills()).to.equal(false);
+    await expect(spokePool.connect(owner).pauseDeposits(true)).to.emit(spokePool, "PausedDeposits").withArgs(true);
+    await expect(spokePool.connect(owner).pauseFills(true)).to.emit(spokePool, "PausedFills").withArgs(true);
+    expect(await spokePool.pausedDeposits()).to.equal(true);
+    expect(await spokePool.pausedFills()).to.equal(true);
+    await expect(spokePool.connect(owner).pauseDeposits(false)).to.emit(spokePool, "PausedDeposits").withArgs(false);
+    await expect(spokePool.connect(owner).pauseFills(false)).to.emit(spokePool, "PausedFills").withArgs(false);
+    expect(await spokePool.pausedDeposits()).to.equal(false);
+    expect(await spokePool.pausedFills()).to.equal(false);
```

Comment:
I suspect that interleaving the test of pausedDeposits and pausedFills hid the error in SpokePool.sol @ line 186. It might be worth splitting the test up so that we are testing each modifier in isolation.

Reply:
you are definitely correct, good reminder, I split this up